themes/default/content/templates/serverless-application/azure/index.md

```diff
@@ -0,0 +1,139 @@
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: serverless-azure
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Azure
+  slug: azure
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The Serverless Application template creates an infrastructure as code project in your favorite language that deploys a serverless application to Azure. It deploys an [Azure Storage Blob account]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccount" >}}) that's configured with a [container for static website hosting]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccountstaticwebsite" >}}) and an [Azure function]({{< relref "/registry/packages/azure-native/api-docs/web/webappfunction" >}}) that runs the business logic, which is written in the same language as the template. The application uses an [App Service plan]({{< relref "/registry/packages/azure-native/api-docs/web/appserviceplan" >}}) by default. The template ships with a placeholder website that displays the current time to give you a working Pulumi project out of the box that you can customize easily and extend to suit your needs.
+
+![An architecture diagram of the Pulumi $CLOUD $ARCHITECTURE template](./architecture.png)
+
+## Using this template
+
+To use this template to deploy your own serverless application, make sure you've [installed Pulumi]({{< relref "/docs/get-started/install" >}}) and [configured your Azure credentials]({{< relref "/registry/packages/azure/installation-configuration#credentials" >}}), then create a new [project]({{< relref "/docs/intro/concepts/project" >}}) using the template in your language of choice:
+
+{{% chooser language "typescript,python,go,csharp" / %}}
+
+{{% choosable language typescript %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-typescript
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-python
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-go
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```bash
+$ mkdir my-serverless-app && cd my-serverless-app
+$ pulumi new serverless-azure-csharp
+```
+
+{{% /choosable %}}
```

Comment:
I can't seem to get GitHub to add this block as a suggestion, but if you copy these lines and add paste them as a new block starting on 73 (so there's an empty line before and after it), then change the two `csharp` references in the new block to `yaml`, you'll get a working YAML option also.